packages/module/src/ListManager/ListManager.tsx

```diff
@@ -0,0 +1,170 @@
+    const newColumns = newOrder.map((item: any) => {
+      const found = currentColumns.find(c => c.key === item.id);
+      if (!found) {
+        throw new Error(`Column with key ${item.id} not found`);
+      }
+      return found;
+    });
+    setCurrentColumns(newColumns);
+    onOrderChange?.(newColumns);
+  };
+
+  const handleSave = () => {
+    onSave?.(currentColumns);
+  };
+
+  const handleBulkSelect = (value: BulkSelectValue) => {
+    const allSelected = value === 'all' || value === 'page';
+    handleSelectAll(allSelected);
+  };
+
+  const handleSelectAll = (select = true) => {
+    const newColumns = currentColumns.map(c => ({ ...c, isSelected: c.isUntoggleable ? c.isSelected : select }));
+    setCurrentColumns(newColumns);
+    onSelectAll?.(newColumns);
+  };
+
+  return (
+    <>
+      <div style={{ paddingBottom: '1rem' }}>
+        <BulkSelect
+          canSelectAll
+          isDataPaginated={false}
+          selectedCount={currentColumns.filter(({ isSelected }) => isSelected).length}
+          totalCount={currentColumns.length}
+          onSelect={handleBulkSelect}
+          pageSelected={currentColumns.every((item) => item.isSelected)}
+          pagePartiallySelected={
+            currentColumns.some((item) => item.isSelected) && !currentColumns.every((item) => item.isSelected)
+          }
+        />
+      </div>
+      <DragDropSort
+        variant="DataList"
+        items={currentColumns.map((column, index) => ({ id: column.key, content: 
+            <DataListItemRow>
+              <DataListCheck
+                data-testid={`column-check-${column.key}`}
+                isChecked={column.isSelected}
+                onChange={() => handleChange(column.key)}
+                isDisabled={column.isUntoggleable}
+                aria-labelledby={`${ouiaId}-column-${index}-label`}
+                ouiaId={`${ouiaId}-column-${index}-checkbox`}
+                id={`${ouiaId}-column-${index}-checkbox`}
+              />
+              <DataListItemCells
+                dataListCells={[
+                  <DataListCell key={column.key} data-ouia-component-id={`${ouiaId}-column-${index}-label`}>
+                    <label htmlFor={`${ouiaId}-column-${index}-checkbox`} id={`${ouiaId}-column-${index}-label`}>
+                      {column.title}
+                    </label>
+                  </DataListCell>
+                ]}
+              />
+            </DataListItemRow>
+        }))}
+        onDrop={onDrag}
+        overlayProps={{ isCompact: true }}
+      >
+        <Droppable 
+          items={currentColumns.map((column) => 
+            // eslint-disable-next-line no-console
+            ({ id: column.key, content: column.title })
+          )} 
+          wrapper={<DataList aria-label="Selected columns" isCompact data-ouia-component-id={`${ouiaId}-column-list`}/>}
+        />
+      </DragDropSort>
+      <ActionList style={{ paddingTop: '1rem' }}>
```

Comment:
can we update these styles to be `style={{ paddingBlockStart: 'var(--pf-t--global--spacer--md)' }}`